### ISSUE TITLE
Possible issue

### ISSUE THREAD

**Issue by midwestfishingadventures** [squizlabs/PHP_CodeSniffer#524]
First here is a code snippet:
```php
$result = $db -> query($sql, $params);
```

Here is the warning and the sniff:
	ObjectOperatorIndent
        Object operator not indented correctly; expected 4 but found 1

I get tons of these types of warnings for instance here is code snippet that generates same warning:
```php
     $params = array(
                              ':linkid' => $this -> id, 
                              ':username' => $this -> changedby,  <==== warning generated on this line
                              ':thedate' => date('Y-m-d G:i:s'), 
                              ':theaction' => 'Add Item', 
                              ':thefields' => '', 
                              ':oldvalues' => '', 
                              ':newvalues' => ''
                          );
```

**Comment by midwestfishingadventures** [comment]
the more I look at this I think it has to do with two object calls in close proximity. For instance:
$this -> id = $db -> getLastInsertId(); generates the same warning.

**Comment by aik099** [comment]
Please use fenced code blocks to enable syntax highlighting.

**Comment by midwestfishingadventures** [comment]
Sorry had to educate myself on how to use the syntax highlighting

Here is another example the error is on the middle line but I think the above line causes it.

```php
$this -> did = $db -> getLastInsertId();
$this -> action = 'Added Document';
$this -> dispatchHistory(ArchiveModel::APP_ID, $this -> did);
```

**Comment by gsherwood** [comment]
I can't get any of this code to generate any errors or warnings.

Are you able to provide the content of a small sample file and the command you are running so I can replicate this issue?

**Comment by midwestfishingadventures** [comment]
Ok I have a small sample file that produces the error.

-bash-4.1$ phpcs checkStyleTest.php

FILE: /opt/html/checkStyleTest.php
--------------------------------------------------------------------------------
FOUND 5 ERROR(S) AFFECTING 4 LINE(S)
--------------------------------------------------------------------------------
  1 | ERROR | End of line character is invalid; expected "\n" but found "\r\n"
  2 | ERROR | Missing file doc comment
  2 | ERROR | Missing class doc comment
  7 | ERROR | Missing comment for param "$types" at position 1
 18 | ERROR | Object operator not indented correctly; expected 16 spaces but
    |       | found 1
--------------------------------------------------------------------------------

Time: 25 ms, Memory: 4.50Mb

-bash-4.1$

the code
```php
<?php
class SurveyPresenter extends CorePresenter
{
    /**
     * themeTypeOptions
     *
     * @param array $types
     *
     * @return string $html
     *
     * @access public
     * @static
     */
    public static function themeTypeOptions($types)
    {
        $html = '';
        foreach ($types as $type) {
            $html .= $type -> tid . '">' . $type -> type;
        }
        return $html;
    }

}
?>
```

**Comment by gsherwood** [comment]
I can't replicate the error with your sample code, but your output tells me that you must be using a fairly old version of PHPCS because you don't have the EOL char error marked as fixable and you don't have a message about upgrading to the 2.x version.

So the first thing to do is try upgrading to the latest stable version because I can't issue a fix for an old version anyway.

**Comment by midwestfishingadventures** [comment]
Yep that was it. Sorry to take so much time.
